content/en/platform/corda/5.0-beta/operating/manual-registration.md

```diff
@@ -0,0 +1,24 @@
+---
+date: '2023-03-15'
+title: "Manual Registration Approval"
+menu:
+  corda-5-beta:
+    identifier: corda-5-beta-manual-registration
+    parent: corda-5-beta-operate
+    weight: 1000
+section_menu: corda-5-beta
+---
+
+Membership groups can be configured to specify that the MGM operator must manually approve (or decline) member registration requests. 
+The approval process can specify that requests satisfying specific criteria require manual approval, while others are approved automatically. 
+The manual approval process presents the request to the MGM operator, enabling the operator to review the request before approving or declining it using the REST API. 
+This process applies to both registration and re-registration requests. 
+The approval process can be configured at any point in time, and only affects future registration requests - previously approved members are not required to re-register.
+
+Registration requests are evaluated according to regular expression-based rules defined by the MGM operator. 
+The proposed [MemberInfo]({{< relref "../developing/api/api-membership.md#memberinfo" >}}) is compared with the previous (if any) `MemberInfo` to calculate the difference in their member contexts. 
```

Comment:
Should MemberInfo here use the same formatting in both cases?

Reply:
I actually logged this to discuss in the next team sync.
Do we have a standard for links with code?

Reply:
Can it be a link within code bars?

Reply:
No, that doesn't work currently 
![image](https://user-images.githubusercontent.com/103633799/225588271-3c3b3dfd-d276-4182-ac65-6e014934f4ec.png)